### ISSUE TITLE
Possibility to inject custom javascript

### ISSUE THREAD

**Issue by mynameisvasco** [tryphotino/photino.NET#58]
The possibility to inject custom javascript would allow us to run custom javascript on external websites. This use case is suitable for things like web automation testing, web scrappers and bots.

**Comment by ottodobretsberger** [comment]
Interesting thought. We will put that on our list of future considerations. There are a lot of security related questions that need to get evaluated and answered.
Changes would need to be made in the .NET and the Native project (separate changes for WebKit and Chromium Edge) to allow these code injections.

**Comment by artemiusgreat** [comment]
Yes, that would be awesome. At the moment, the only way to interact with an external webpage is by using Chrome Extensions or Web Drivers, like Selenium, Puppeteer, etc. Photino could be a much simpler option.

**Comment by rtmann** [comment]
Not an answer for Photino, but if you are ok with using Windows you can use CefSharp to run a full fledged Chrome Browser embeded in your application and you can do EVERYTHING with that thing.  You can intercept cookies, inject cookies, intercept page loads/replace script links, inject JS, and on and on and on.

I actually build an SPA crawler using it a few years ago to index spa's.
